Makefile

```diff
@@ -0,0 +1,6 @@
+
+unittest:
+	python -m pytest tests
+
+package:
+	python -m build --sdist --wheel --outdir dist/.
```

Comment:
Maybe better if we use `python3`

We need to add `build` module into requirements.txt

Reply:
Changed to python3.
build already in requirements.txt.